easygui/boxes/choice_box.py

```diff
@@ -239,8 +233,9 @@ def get_pos(self):
         global_state.window_position = '+' + geom.split('+', 1)[1]
 
     def preselect_choice(self, preselect):
-        self.choiceboxWidget.select_set(preselect)
-        self.choiceboxWidget.activate(preselect)
+        if preselect <> None:
```

Comment:
I am not clear on the python difference between "<> None" and "is not None".  I like the "is not None" because it is closer to the database phrasing of "is not null".